lessons/prework/README.md

```diff
@@ -0,0 +1,21 @@
+# Unit 0: Prework
```

Comment:
Yeah units are - but this is not really a standalone unit. Would you rather include it as part of Unit 1? Or just not have a number? Don't think it'd be worth bumping up all the other unit #s just for this